cfgov/unprocessed/js/organisms/FeaturedContentModule.js

```diff
@@ -22,46 +19,52 @@ let _videoPlayer;
  * @returns {FeaturedContentModule} An instance.
  */
 function FeaturedContentModule( element ) {
-  _dom = checkDom( element, BASE_CLASS );
+  const _dom = checkDom( element, BASE_CLASS );
 
-  this.init = init;
+  let _videoPlayer = null;
 
-  return this;
-}
+  /**
+   * @returns {FeaturedContentModule|undefined} An instance.
+   *   or undefined if it was already initialized.
+   */
+  function _init() {
+    if (!setInitFlag( _dom ) ) {
+      let UNDEFINED;
+      return UNDEFINED;
+    }
 
-/**
- * @returns {FeaturedContentModule} An instance.
- */
-function init() {
-  setInitFlag( _dom );
+    const videoPlayerDom = _dom.querySelector( `.${ VideoPlayer.BASE_CLASS }` );
+
+    // If we don't have a video in this FCM, bail out.
+    if ( videoPlayerDom === null ) {
+      return this;
+    }
 
-  const videoPlayerDom = _dom.querySelector( `.${ VideoPlayer.BASE_CLASS }` );
+    _videoPlayer = new VideoPlayer( videoPlayerDom );
+    _videoPlayer.addEventListener( 'onPlay', _videoPlayHandler.bind( this ) );
+    _videoPlayer.addEventListener( 'onStop', _videoStopHandler.bind( this ) );
+    _videoPlayer.init();
 
-  // If we don't have a video in this FCM, bail out.
-  if ( videoPlayerDom === null ) {
     return this;
   }
 
-  _videoPlayer = new VideoPlayer( videoPlayerDom );
-  _videoPlayer.addEventListener( 'onPlay', _videoPlayHandler );
-  _videoPlayer.addEventListener( 'onStop', _videoStopHandler );
-  _videoPlayer.init();
+  /**
+   * Event handler for when video has begun playing.
+   */
+  function _videoPlayHandler() {
+    _dom.classList.add( 'video-playing' );
+  }
 
-  return this;
-}
+  /**
+   * Event handler for when video has stopped playing.
+   */
+  function _videoStopHandler() {
+    _dom.classList.remove( 'video-playing' );
+  }
 
-/**
- * Event handler for when video has begun playing.
- */
-function _videoPlayHandler() {
-  _dom.classList.add( 'video-playing' );
-}
+  this.init = _init;
```

Comment:
`_init` is being set to public here so shouldn't have an underscore prefix.

Reply:
Thanks, I made similar changes to the public methods in VideoPlayer as well.